modules/installation-gcp-config-yaml.adoc

```diff
@@ -17,6 +17,11 @@ ifeval::["{context}" == "installing-gcp-private"]
 :private:
 :vpc:
 endif::[]
+ifeval::["{context}" == "installing-gcp-user-infra-vpc"]
```

Comment:
Regarding this: 
To use an existing VPC in a cluster that uses infrastructure that you provision, you must set publish to Internal

I suggest to replace "an existing" with "a shared". Shared VPC stands for cross project VPC which is not quite same as existing VPC